assets/src/edit-story/components/panels/panel/simplePanel.js

```diff
@@ -28,8 +28,8 @@ import PanelContent from './shared/content';
 
 function SimplePanel({ children, name, title, ...rest }) {
   return (
-    <Panel name={name} {...rest}>
-      <PanelTitle>{title}</PanelTitle>
+    <Panel name={name}>
```

Comment:
Why was this changed? Some panels rely on these props being passed down.

Change looks unrelated.

Reply:
It was mistakenly deleted from resolving conflict. Added back.